pages/research_software/life_cycle.md

```diff
@@ -11,7 +11,7 @@ This lifecycle was developed as part of the EOSC Task Force [Infrastructure for
 
```

Comment:
Line 9 still says "Tier 2 or 3", which should now become "Tier 1 or 2".

Reply:
I'll fix that and update - thanks for the catch!